docs/set_up_admin_tails.rst

```diff
@@ -61,55 +61,70 @@ Start by running the following commands to download the git repository.
 Verify the Release Tag
 ~~~~~~~~~~~~~~~~~~~~~~
 
-.. caution:: Do not skip this step as this steps validates the files
-             in your Git repository.
+.. important::
 
-First, download and verify the **SecureDrop Release Signing Key**.
+   It is crucial for the integrity of your installation that you carefully
+   follow the instructions below. By following these steps, you will verify
+   if your copy of the codebase has been approved by the SecureDrop
+   development team.
+
+Download and verify the **SecureDrop Release Signing Key** using the following
+command:
 
 .. code:: sh
 
-    gpg --recv-key "2224 5C81 E3BA EB41 38B3 6061 310F 5612 00F4 AD77"
+   gpg --keyserver hkps://keys.openpgp.org --recv-key \
+    "2224 5C81 E3BA EB41 38B3 6061 310F 5612 00F4 AD77"
+
+If you are not copy-pasting this command, we recommend you double-check you have
+entered it correctly before pressing enter. GPG will implicitly verify that the
+fingerprint of the key received matches the argument passed.
+
+.. _Tails is connected to Tor: https://tails.boum.org/doc/anonymous_internet/tor_status/index.en.html
 
-.. note:: It is important you type this out correctly. If you are not
-          copy-pasting this command, we recommend you double-check you have
-          entered it correctly before pressing enter.
+If GPG warns you that the fingerprint of the key received does not
+match the one requested, do **not** proceed with the installation. If this
+happens, please contact us at securedrop@freedom.press.
 
-.. tip:: If the ``--recv-key`` command fails, first double-check that
-   `Tails is connected to Tor`_.
+.. note::
 
-   Once you've confirmed that you're successfully connected to Tor, try
-   re-running the ``--recv-key`` command a few times. The default GPG
-   configuration on Tails uses a keyserver pool, which may occasionally return
-   a malfunctioning keyserver, causing the ``--recv-key`` command to fail.
+   If the ``--recv-key`` command fails, first double-check that
+   `Tails is connected to Tor`_. Once you've confirmed that you're successfully
+   connected to Tor, try re-running the ``--recv-key`` command a few times.
 
-   If the command is consistently failing after a few tries, it could
-   indicate that the default GPG key servers are down or unreachable. As a
-   workaround, another keyserver can be specified by adding the ``--keyserver``
-   option to the ``gpg --recv-key`` command. In our experience, the SKS HKPS
-   keyserver pool is usually a reliable alternative, so try:
+   If the command still fails, the *keys.openpgp.org* keyserver may be down.
+   In that case, we recommend downloading the key from the SecureDrop website:
 
    .. code:: sh
 
-      gpg --keyserver hkps://hkps.pool.sks-keyservers.net --recv-key "2224 5C81 E3BA EB41 38B3 6061 310F 5612 00F4 AD77"
+      cd ~/Persistent
+      torify curl -LO https://securedrop.org/securedrop-release-key.asc
```

Comment:
Note: This redirect has never been used before; I created it today (using Wagtail's redirect feature), as the official URL (https://securedrop.org/documents/10/securedrop-release-key.asc) is more difficult to type reliably, and subject to change if a new version is uploaded (in which case we'd have to redirect anyway, as we're doing from https://securedrop.org/documents/1/securedrop-release-key.asc). Happy to use an alternative approach if preferable.